frontend/schema/class-schema-person.php

```diff
@@ -156,18 +155,57 @@ protected function build_person_data( $user_id ) {
 	 * @return array $data The Person schema.
 	 */
 	protected function add_image( $data, $user_data ) {
+		$schema_id = $this->context->site_url . WPSEO_Schema_IDs::PERSON_LOGO_HASH;
+
+		$data = $this->set_image_from_options( $data, $schema_id );
+		if ( isset( $data['image'] ) ) {
+			return $data;
+		}
+
+		$show_avatars = get_option( 'show_avatars' );
```

Comment:
Please confirm that this check shouldn't precede the above `if ( isset( $data['image'] ) ) {`

Reply:
Yeah we only check this here, because we only fall back to an avatar if we don't have an image in our settings.